packages/module/src/InvalidObject/InvalidObject.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Title } from '@patternfly/react-core';
+
+import Icon404 from './icon-404';
+import React from 'react';
+import classNames from 'classnames';
+import { createUseStyles } from 'react-jss'
+
+const useStyles = createUseStyles({
+  "invalidObject": {
```

Comment:
```suggestion
  invalidObject: {
```

Reply:
I don't think the styling is needed with the EmptyState, it works even without it

Reply:
Yeah I got rid of it. It does work without it.